src/main/java/edu/umich/tl/CourseDelete.java

```diff
@@ -169,69 +178,70 @@ private static void checkForApiType(Properties properties) {
 				} 
 			}
 		} catch (JsonParseException e) {
```

Comment:
In Java 1.7 and later the catch block have a list of exceptions to handle. If you can get the exception type from the exception object you could cut down the number of catch blocks.  (We shouldn't be using Java 1.6 anymore.)